Taskfile.yml

```diff
@@ -281,3 +286,20 @@ tasks:
     cmds:
       - task: source:build
       - task: source:push
+
+  translations:scan:
+    desc: Scan and generate translations
+    cmds:
+     # Generate translations for custom modules
+      - task dev:cli -- drush potion:generate $LANGUAGE modules/custom/ {{ .PO_DIR }} --recursive --default-write-mode=override
+    # Generate translations for custom themes
+      - task dev:cli -- drush potion:generate $LANGUAGE themes/custom/ {{ .PO_DIR }} --recursive --default-write-mode=merge
+    # Fill translations from db
+      - task dev:cli -- drush potion:fill $LANGUAGE {{ .PO_DIR }}/$LANGUAGE.po
+    # We need to chown the PO file in the container with the user of the host
+    # in order to be able to commit the file afterwards.
+    # NB: Using a fixed UID/GID might gives us problems in the future.
+      - task dev:cli -- chown 1001:1001 web/{{ .PO_DIR }}/$LANGUAGE.po
```

Comment:
The uid is not just grabbed out of the thin air :) It is the uid of the user running the GH action container.
I'll try to find a more future proff solution. Otherwise we have to go for this for now :)

Reply:
Using `id` cmd now. It should be future proof (dangerous word though)